Gemfile

```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
+# rails-assets requires >= 1.8.4
+gem 'bundler', '>= 1.8.4'
```

Comment:
This will only install bundler in the bundle but not force the version it's built with :-)